### DISCUSSION TITLE
Demo: Web3auth mpc demo webapp is not working

### DISCUSSION THREAD

**Discussion by nlok5923** [Web3Auth/Web3Auth#1267] [Help]
Tried google login over it and received this:

demo: https://demo-mpc.web3auth.io/

![image](https://user-images.githubusercontent.com/56120084/216026466-f58d560e-a142-4e78-ab24-1851eee3999e.png)

**Comment by arch1995** [comment]
Hi @nlok5923 

This has been fixed. Please try again.

**Reply by nlok5923** [reply]
Hi Thanks for replying.

It worked partially but stucked at precompute.
![image](https://user-images.githubusercontent.com/56120084/216669418-62b26870-312e-43f9-84e1-f25265475189.png)

**Reply by arch1995** [reply]
Please try again.

**Reply by nlok5923** [reply]
Thanks @arch1995 for fixing! It works now
